axiom/util_object.c

```diff
@@ -1906,3 +1906,44 @@ char* nro_dump(const nrobj_t* obj) {
 
   return str;
 }
+
+static char* stringify_boolean(const nrobj_t* obj) {
+  nr_status_t err;
+  nrbuf_t* buf;
+  char* ret;
+  int ival = nro_get_ival(obj, &err);
+
+  buf = nr_buffer_create(1024, 1024);
```

Comment:
Is there a good way to do this function without a heap allocation here?

Reply:
Mostly I did it this way to mimic the existing program flows with nrbuf_t + supporting functions -> nr_strdup() but I can probably cut all that out in favor of a simpler approach

Reply:
I'm going to leave this as it is, since the flow mimics the existing program convention and the impact is limited exclusively to a handful of tests requiring the boolean conversion, and doesn't have any exposure to core agent functionality